.github/ISSUE_TEMPLATE/3-performance-issue-report.yml

```diff
@@ -0,0 +1,68 @@
+name: 🔧 Performance issue report
+description: Report a performance issue if something isn't performing well in WooCommerce Core.
+title: "[Performance]: "
+labels: [ "type: enhancement", "focus: performance"]
+projects: ["woocommerce/220"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide us with the information requested in this report. 
+        Without these details, we won't be able to evaluate this issue fully.
+        Reports lacking detail or for any other reason than to report a performance issue may be closed without action.
+
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm these before submitting the issue.
+      options:
+        - label: I have carried out troubleshooting steps and I believe I have identified a performance issue.
+        - label: I have searched for similar bugs in both open and closed issues and cannot find a duplicate.
+    validations:
+      required: true
+
+  - type: textarea
```

Comment:
Here and below: aligned with p1736524834757259-slack-C07JJG089M0 and recently created spikes.